### PR TITLE
add internal link documentation

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: en/user_guide/internal_link.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Internal Link
+=============
+
+Open Orchestra use TinyMce to editing rich text.
+A custom button allows to add an internal link, ie a link to a page managed by the Back Office.
```
Comment (ngilain) [inline en/user_guide/internal_link.rst]: `a custom button allows` is weird in this context, I'd rather say `a custom button is used to` here

File: en/user_guide/internal_link.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+.. image:: ../../images/tiny_internal_link.png
+
+This button opens a modal with a form to construct the internal link:
+.. image:: ../../images/form_internal_link.png
+
+The form contains :
+
+* Label: the label for the link displayed in Front Office
+* Pages: the target of the link
+* Content: as specified in the `contents display`_, content blocks need content id in the url. If the target of the link contains such a block, you can choose the id with this sub form. To simplify the choice of the id, the content form contains search filters as content type and keywords (see `boolean expression`_). A refresh button allow to see filtered contents.
```
Comment (ngilain) [inline en/user_guide/internal_link.rst]: the content display documentation

**Comment by veolys-nanne** [comment]
done @itkg-ngilain

File: en/user_guide/internal_link.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
+Internal Link
+=============
+
+Open Orchestra use TinyMce to editing rich text.
```
Comment (ngilain) [inline en/user_guide/internal_link.rst]: to edit

File: en/user_guide/internal_link.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+This button opens a modal with a form to construct the internal link:
+.. image:: ../../images/form_internal_link.png
+
+The form contains :
+
+* Label: the label for the link displayed in Front Office
+* Pages: the target of the link
+* Content: as specified in the `contents display documentation`_, content blocks need content id in the url. If the target of the link contains such a block, you can choose the id with this sub form. To simplify the choice of the id, the content form contains search filters as content type and keywords (see `boolean expression`_). A refresh button allow to see filtered contents.
+* Site: The site target of the link as Open Orchestra is multi-site. A refresh button allow to see site aliases of the choosen site and optionnaly specify one.
```
Comment (ngilain) [inline en/user_guide/internal_link.rst]: I don't like "allow" everywhere, it's not a matter of allowing or denying an access ^^ What about something like "When clicked, the refresh button update the site aliases [...]"

**Comment by ngilain** [comment]
:+1: with or wihout the 2nd modification

**Comment by veolys-nanne** [comment]
for validation, @itkg-rcarles

**Comment by raphaelcarles** [comment]
@itkg-nanne : une choise n'est pas claire : tous les champs se cumulent pour générer 1 url ou certains champs prennent le pas sur les autres (en gros le renseignement d'un champ remplace les autres champs) ?

**Comment by veolys-nanne** [comment]
@itkg-rcarles : as said in documentation, "This button opens a modal with a form to construct the internal link". The fields are cumulative and no replacement is done. As it is the default behavior, should I add something to the documentation?

**Comment by raphaelcarles** [comment]
:+1:  ok

**Pull request merged**